src/lib/cache.js

```diff
@@ -19,6 +20,8 @@ const isTest = NODE_ENV === "test"
 
 const VerboseEvents = ["issue", "failure", "reconnecting", "reconnect", "remove"]
 
+export const cacheEnabled = CACHE_ENABLED === "true"
```

Comment:
It isn’t idiomatic to have camelCase constants. How about defining `CACHE_ENABLED` as a boolean in the `config` module instead of defining `cacheEnabled` at all?

Reply:
Going to switch it to `CACHE_DISABLED` in that case as I want it to be by default, enabled